### PR TITLE
SwitchApp feature

### PR DESCRIPTION
**Pull request by klesh**
Bring application to front if it's already running

### CODE REVIEW

File: org.albert.extension.external.switchapp.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+if albert_op == 'METADATA':
+    metadata = """{
+    "iid": "org.albert.extension.external/v2.0",
+    "name": "WindowSwitcher",
+    "version": "1.0",
+    "author": "Klesh Wong",
+    "dependencies": [],
+    "trigger": "`"
+    }"""
+    print(metadata)
```
Comment (idkCpp) [inline org.albert.extension.external.switchapp.py]: I think I have to make a wiki article about this.
Please make the metadata a dict as well. It improves maintainability and syntax highlighting

Comment (klesh) [inline org.albert.extension.external.switchapp.py]: Is there any example for dependencies checking?
Which section is the place to do the checking?
What is the format should I return for unmet dependencies?

Comment (idkCpp) [inline org.albert.extension.external.switchapp.py]: The initialize section is responsible for the checking. Any output during initialize its treated as error message.

Comment (klesh) [inline org.albert.extension.external.switchapp.py]: I tried it a little bit, 

elif albert_op == "INITIALIZE":
    try:
        process = subprocess.Popen(['wmctrl1', '-lx'], stdout=subprocess.PIPE)
        system.exit(0)
    except Exception, e:
        print(e)
        sys.exit(1)

but it didn't seem to be working, I got no message from Albert...

Comment (idkCpp) [inline org.albert.extension.external.switchapp.py]: With the last version the error was displayed in the settings widget.
Currently I have no clue how this is propagated.
I assume @ManuelSchneid3r knows more on this.

Comment (klesh) [inline org.albert.extension.external.switchapp.py]: @idkCpp I tested it, the plugin will disappear from Settings / Plugins / External extensions, and no message will be displayed to user. I don't think it's a good idea, user will never know why the plugin doesn't show up.
How about I return a dummy item with error message for ALBERT_QUERY ?

Comment (idkCpp) [inline org.albert.extension.external.switchapp.py]: The dummy item would be an option but in my opinion albert needs to be able to handle these kinds of error messages... I'll have a look into it...

Comment (klesh) [inline org.albert.extension.external.switchapp.py]: OK

**Comment by ManuelSchneid3r** [comment]
How does this differ from  #12?

**Comment by ManuelSchneid3r** [comment]
Closing since this extension is deprecated.

**Comment by ManuelSchneid3r** [comment]
However note the poll [here](https://github.com/albertlauncher/albert/issues/123#issuecomment-413868346)

**Pull request closed without merging**